src/couch/src/couch_doc.erl

```diff
@@ -116,14 +131,23 @@ to_json_attachments(Atts, OutputData, Follows, ShowEnc) ->
 to_json_obj(Doc, Options) ->
     doc_to_json_obj(Doc, Options).
 
-doc_to_json_obj(#doc{id=Id,deleted=Del,body=Body,revs={Start, RevIds},
-            meta=Meta}=Doc,Options)->
-    {[{<<"_id">>, Id}]
-        ++ to_json_rev(Start, RevIds)
-        ++ to_json_body(Del, Body)
-        ++ to_json_revisions(Options, Start, RevIds)
-        ++ to_json_meta(Meta)
-        ++ to_json_attachments(Doc#doc.atts, Options)
+doc_to_json_obj(
```

Comment:
This is verbose but this code violates CouchDB formatting rules where we try to not match too many arguments in the function clause header. We supposed to write the following here:
```
doc_to_json_obj(revs={Start, RevIds}, Options) ->
    #doc{
            id = Id,
            deleted = Del,
            body = Body,
            meta = Meta
     } = Doc,
     ...
```

0